shell/common/node_bindings.cc

```diff
@@ -370,6 +330,52 @@ bool NodeBindings::IsInitialized() {
   return g_is_initialized;
 }
 
+// Initialize Node.js cli options to pass to Node.js
+// See https://nodejs.org/api/cli.html#cli_options
+void NodeBindings::SetNodeCliFlags() {
+  const std::unordered_set<base::StringPiece, base::StringPieceHash> allowed =
+      GetAllowedDebugOptions();
+
+  const auto argv = base::CommandLine::ForCurrentProcess()->argv();
+  std::vector<std::string> args;
+
+  // TODO(codebytere): We need to set the first entry in args to the
+  // process name owing to src/node_options-inl.h#L286-L290 but this is
+  // redundant and so should be refactored upstream.
+  args.reserve(argv.size() + 1);
+  args.emplace_back("electron");
+
+  for (const auto& arg : argv) {
+#if BUILDFLAG(IS_WIN)
+    const auto& option = base::WideToUTF8(arg);
+#else
+    const auto& option = arg;
+#endif
+    const auto stripped = base::StringPiece(option).substr(0, option.find('='));
+
+    // Only allow in no-op (--) option or DebugOptions
+    if (allowed.count(stripped) != 0 || stripped == "--")
+      args.push_back(option);
+  }
+
+  // We need to disable Node.js' fetch implementation to prevent
+  // conflict with Blink's.
+  if (browser_env_ != BrowserEnvironment::kBrowser) {
```

Comment:
```suggestion
  if (browser_env_ == BrowserEnvironment::kRenderer ||
      browser_env_ == BrowserEnvironment::kWorker) {
```

To support the defaults in all non-renderer processes (ex: utility)